src/structures/Webhook.js

```diff
@@ -180,9 +180,8 @@ class Webhook {
         auth: false,
       })
       .then(d => {
-        const channel = this.client.channels ? this.client.channels.cache.get(d.channel_id) : undefined;
-        if (!channel) return d;
-        return channel.messages.add(d, false);
+        const channel = this.client.channels?.cache.get(d.channel_id);
+        return channel?.messages.add(d, false) ?? d;
```

Comment:
Why not combine those?

Reply:
oversight

Reply:
More than an oversight, it's related to the previous code, which was `channels ? channels.messages.add(d, false) : d`. Just needed a second refactor pass 😄